examples/express/package.json

```diff
@@ -10,8 +10,8 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@react-router/express": "7.6.0",
-    "@react-router/node": "7.6.0",
```

Comment:
I know this is the example package, but is there any reason to keep this pinned as well?